yb-voyager/cmd/exportData.go

```diff
@@ -1517,9 +1518,37 @@ func startFallBackSetupIfRequired() {
 	if utils.DoNotPrompt {
 		cmd = append(cmd, "--yes")
 	}
-	if disablePb {
+
+	passImportDataToSourceSpecificCLIFlags := map[string]bool{
+		"disable-pb": true,
+	}
+
+	// Check whether the command specifc flags have been set in the config file
+	if cfgFile != "" {
```

Comment:
It would be good to isolate all viper code only in the root command pre-run step. i.e. the first thing that we do is extract all the information that we want from viper and store it somewhere. Rest of the code only works upon those stored variables, and does not interact with viper directly. 

So, how about we capture all these flag values in `initConfig` and store them somewhere, which can then be used by this function subsequently?

Reply:
Yep done that. You can have a look.